### DISCUSSION TITLE
Question about installing app in production

### DISCUSSION THREAD

**Discussion by john-kuo** [kinngh/shopify-node-express-mongodb-app#125] [Q&A] [answered]
Hey @kinngh ,

I submitted my app to Shopify and they came back saying they couldn't go through the OAuth process.  I realized it's because I deployed the app in dev mode, so they couldn't install it directly in Parner's dashboard.

Just redeployed the server with with **npm start**, as to the client portion, do I use the **dev:client** or just **build** ?
When I tried to install it from the Partner's dashboard it gave me this error 
<img width="1285" alt="Screenshot 2024-04-17 at 4 28 07 PM" src="https://github.com/kinngh/shopify-node-express-mongodb-app/assets/1480108/f5700764-d823-44f8-83c7-2df9bd689892">

thanks!

**Comment by kinngh** [comment, marked as answer]
Ok I'm a little surprised that this is even a question but I'm just gonna assume this is your first time. To deploy, first `npm run build` and then `npm run start`. ABSOLUTELY NOTHING RUNS IN DEV MODE. Aight?

**Reply by john-kuo** [reply]
Ahh, it was a misunderstanding..  The review submission required a tutorial video.  I gave them the installation link with the dev deployment link like this https://host/api/auth?shop={shopName} .  Turned out they just clicked on the install button in the panel, that's why that 404 showed up.  And I thought maybe I missed out on something else..  , but yeah, its been resolved, I deployed the production build and it would install properly when clicking on the Install link in the Partner's panel.

Thanks again man
